src/anchorfm-pupeteer/index.js

```diff
@@ -200,6 +187,33 @@ function getSaveDraftOrPublishOrScheduleButtonDescription() {
   };
 }
 
+
+async function clickSelector(page, selector, {visible} = {}) {
```

Comment:
II would advise the third argument to be an "options" argument.

So make the third argument options.